.github/workflows/integration_tests.yml

```diff
@@ -492,214 +474,62 @@ jobs:
-        timeout-minutes: 5
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run LogForwarding -v
-
-      - name: Test Sequence States
-        # do not run this test for the airgapped scenario
-        if: env.RUN_AIRGAPPED_TEST != 'true'
-        id: test_sequence_states
-        timeout-minutes: 5
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run SequenceState -v
-
-      - name: Test Sequence Loop
-        # do not run this test for the airgapped scenario
-        if: env.RUN_AIRGAPPED_TEST != 'true'
-        id: test_sequence_loop
-        timeout-minutes: 5
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run SequenceLoopIntegrationTest -v
-
-      - name: Test Quality Gates Standalone
-        id: test_quality_gates
-        # Note: quality gates test requires network access, which doesn't work on some platforms (also not on airgapped)
-        if: env.RUN_AIRGAPPED_TEST != 'true' && env.RUN_QUALITY_GATES_TEST == 'true'
-        timeout-minutes: 10
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run Test_QualityGates -v
-
-      - name: Test Self Healing
-        # do not run this test for the airgapped scenario
-        if: env.RUN_AIRGAPPED_TEST != 'true'
-        id: test_self_healing
-        timeout-minutes: 10
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-          UNLEASH_SERVICE_VERSION: "release-0.3.2"
-        run: cd test/go-tests && go test -run Test_SelfHealing -v
-
-      - name: Test Delivery Assistant
-        id: test_delivery_assistant
-        # Delivery Assistant only works when we install keptn with continuous delivery use-case
-        if: env.RUN_AIRGAPPED_TEST != 'true' && env.RUN_CONTINUOUS_DELIVERY_TEST == 'true'
-        timeout-minutes: 7
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run Test_DeliveryAssistant -v
-
-      - name: Test Backup and Restore
-        id: test_backup_restore
-        if: env.BACKUP_RESTORE == 'true'
-        timeout-minutes: 7
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run Test_BackupRestore -v
+      - name: Prepare test run
+        id: prepare_test_run
+        run: |
+          go install gotest.tools/gotestsum@latest
```

Comment:
I would pin it to a specific version to avoid flakiness due to a breaking change

Reply:
good catch!